.github/workflows/dev-deploy-dashboard.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy DEV DASHBOARD
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+      - main
+    paths:
+      - 'apps/web/**'
+      - 'apps/dashboard/**'
```

Comment:
We need to deploy to dev when any changes in the web or dashboard apps. But I'm wondering like whether we should include all the web app deps, like shared, maybe inbox etc?